students/ReemAlqaysi/Lesson04/mailroom.py

```diff
@@ -0,0 +1,110 @@
+"""
+MAIL ROOM PART 1
+this script has a data structure that holds a list of  donors and a history of the amounts they have donated. 
+This structure should be populated at first with at least five donors, with between 1 and 3 donations each.
+ data structure stored in the global namespace.
+The script prompts the user (you) to choose from a menu of 3 actions: 
+“Send a Thank You”, “Create a Report” or “quit”.
+"""
+
+from operator import itemgetter
+import sys  # imports go at the top of the file
+
+#initial donations ammount and givers names.
+donor_list = {
+    "Jan Balard": (600.00,250.00),
+    "Joe McHennry": (1500.00,1500.00),
+    "Jeff Hansen": (450.00,150.00),
+    "Scott Newman": (100.00,5000.00),
+    "Rabi Das": (500.00,950.00)
```

Comment:
this should not be a tuple since tuple are immutable and you do want to change this to add more donations.